README.md

```diff
@@ -0,0 +1,128 @@
+# Mountsion CBS Secondary Website
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. Here's how you can get set up
+
+-First head to the repository, located at https://github.com/MountSionCBSSecondary/SchoolWebsite
+
+-Then clone our repostory
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Screenshot from 2018-04-17 12-12-37.png)
+
+First you're going to click on "Clone or Download"
+
+Then proceed to copy the link
+
+---
+
+Now open your terminal
+
+Type
+`git clone <url>`
+
+### Terminal
+
+Wherever you are in terminal is where your file will be saved. TO navigate through the terminal you will only need 3 simple commands
+
+`cd`
+
+`ls`
+
+`cd..`
+
+The `cd` command is for changing directories, it is used by typing `cd <name-of-file>`
+
+The `ls` command will show all the files in the current directory, this makes it easy to navigate between files
+
+the `cd..` command will go back a directory, if you accidentally go to the wrong file you can use this to go back
+
+### Prerequisites
```

Comment:
As above this section can be removed, feel free to add them to the tutorial doc, and make a separate PR for it